docs/api-docs/index.md

```diff
@@ -7,9 +7,37 @@ Macrometa's GDN service offering is API-first. We create API calls for every tas
 We currently have two ways in which you can interact with the API: the app API reference and the docs API reference.
 The first way of using the REST API is the API reference in the GDN web browser interface (app API). Use the built-in API reference to run various calls and view their input and output.
 
-While you can run your API calls from our docs API reference, the second way of using the REST API is through your favorite IDE, terminal, or other interfaces (docs API). 
+While you can run your API calls from our docs API reference, the second way of using the REST API is through your favorite IDE, terminal, or other interfaces (docs API).
 
 In this section, pages describe how to:
 
 - [Run API commands from the Macrometa app API reference](run-api-commands-app.md).
 - [Run API commands from the Macrometa documentation API reference](run-api-commands-docs.md).
+
+## API Authorization
+
+You can use JSON web tokens (JWTs) or API keys to authorize API calls. We recommend using API keys.
+
+To authorize your API call, you must prepend the appropriate prefix to your authorization token:
+
+- JWT: `bearer <your-jwt>`
+- API key: `apikey <your-api-key>`
+
+## Unique Global URL vs. Generic
+
+If you have a Macrometa Play account, then you can use the generic API server address: `https://api-play.paas.macrometa.io`. This address is used in most of our tutorials and getting started examples in the documentation.
+
+However, for production apps, or any account that is not on the Play tier, you must use your unique global URL.
+
+To find your unique API URL:
+
+1. In your Macrometa account, switch to the geofabric that you want to run API calls against. This is the fabric that contains the collections, streams, or other entities that are the target of your API calls.
```

Comment:
```suggestion
1. In your Macrometa account, switch to the GeoFabric that you want to run API calls against. This is the fabric that contains the collections, streams, or other entities that are the target of your API calls.
```

We've been referring to geofabrics as "GeoFabrics," but that decision was made a while ago.  Off the top of my head i'm not sure how (or if) GeoFabrics are referred to in the GUI.

Reply:
In the GUI, they are Geo Fabrics. Ugh, drives me nuts.